### PR TITLE
:seedling: Add release 1.8 to the test matrix

### PR DESCRIPTION
**Pull request by mquhuy**
Add CAPM3 and IPAM release-1.8 + BMO release-0.8 to the test matrix.

Also putting release-1.6 and BMO release-0.5 to `Tested`.

### CODE REVIEW

File: docs/user-guide/src/version_support.md [hunk 1]
```diff
@@ -102,16 +105,16 @@ The table describes which branches/image-tags are tested in each periodic CI tes
...
 | metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
 | metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
-| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.5    | v0.5.0         | latest      | v23.1.0    |
 
 | FEATURE AND E2E TESTS                                          | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
 | -------------------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | metal3-periodic-ubuntu/centos-e2e-feature-test-main            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8     | release-1.8  | release-1.8 | release-0.7    | v0.7.0         | latest      | v25.0.0    |
```
Comment (smoshiur1237) [inline docs/user-guide/src/version_support.md:115]: I missed this to change, test triggers are not valid anymore
```
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-pivoting
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-remediation
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-features
```
same goes for other branch.

Comment (mquhuy) [reply docs/user-guide/src/version_support.md]: @smoshiur1237 So the `periodic-` part was removed from the names, right?

File: docs/user-guide/src/version_support.md [hunk 2]
```diff
@@ -100,18 +103,20 @@ The table describes which branches/image-tags are tested in each periodic CI tes
 
 | INTEGRATION TESTS                                              | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
 | -------------------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
```
Comment (tuminoid) [inline docs/user-guide/src/version_support.md:105]: Make header width match the content.

File: docs/user-guide/src/version_support.md [hunk 3]
```diff
@@ -100,18 +103,20 @@ The table describes which branches/image-tags are tested in each periodic CI tes
...
+| metal3-ubuntu/centos-e2e-integration-test-main        | main         | main        | main           | latest         | latest      | latest     |
+| metal3_main_integration_test_ubuntu/centos            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-8 | release-1.8  | release-1.8 | release-0.7    | v0.7.0         | latest      | v25.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.4    | v0.5.0         | latest      | v23.1.0    |
 
 | FEATURE AND E2E TESTS                                          | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
```
Comment (tuminoid) [inline docs/user-guide/src/version_support.md:113]: Same in this table.

**Review by tuminoid — COMMENTED** [review]
/lgtm

File: docs/user-guide/src/version_support.md [hunk 4]
```diff
@@ -98,20 +101,22 @@ The table describes which branches/image-tags are tested in each periodic CI tes
...
+| INTEGRATION TESTS                                     | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
+| ----------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
+| metal3-ubuntu/centos-e2e-integration-test-main        | main         | main        | main           | latest         | latest      | latest     |
+| metal3_main_integration_test_ubuntu/centos            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-8 | release-1.8  | release-1.8 | release-0.8    | v0.8.0         | latest      | v25.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.4    | v0.5.0         | latest      | v23.1.0    |
```
Comment (smoshiur1237) [inline docs/user-guide/src/version_support.md:111]: why did you remove the periodic part from the test trigger? These triggers are not valid.
```suggestion
| metal3-periodic-ubuntu/centos-e2e-integration-test-main        | main         | main        | main           | latest         | latest      | latest     |
| metal3_periodic_main_integration_test_ubuntu/centos            | main         | main        | main           | latest         | latest      | latest     |
| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-8 | release-1.8  | release-1.8 | release-0.8    | v0.8.0         | latest      | v25.0.0    |
| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.4    | v0.5.0         | latest      | v23.1.0    |
```

File: docs/user-guide/src/version_support.md [hunk 5]
```diff
@@ -98,20 +101,22 @@ The table describes which branches/image-tags are tested in each periodic CI tes
...
+
+| FEATURE AND E2E TESTS                                 | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
+| ----------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
+| metal3-ubuntu/centos-e2e-feature-test-main            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-8     | release-1.8  | release-1.8 | release-0.8    | v0.8.0         | latest      | v25.0.0    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-7     | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-6     | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-5     | release-1.5  | release-1.5 | release-0.4    | v0.4.0         | latest      | v23.1.0    |
```
Comment (smoshiur1237) [inline docs/user-guide/src/version_support.md:119]: Same here, please add periodic. 
Also these are not correct trigger as those are divided in three parts. pivoting, remediation and features. we don't run any ubuntu feature test in CI periodics, only centos.
```
metal3-periodic-centos-e2e-feature-test-main-pivoting
metal3-periodic-centos-e2e-feature-test-main-remediation
metal3-periodic-centos-e2e-feature-test-main-features 
```
jenkins: https://jenkins.nordix.org/view/Metal3%20Periodic/

**Review by smoshiur1237 — COMMENTED** [review]
Added comment to update the trigger

File: docs/user-guide/src/version_support.md (hunk 1 above)
Comment (smoshiur1237) [reply docs/user-guide/src/version_support.md]: Periodic part was not removed

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by smoshiur1237 — COMMENTED** [review]
/approve

**Review by kashifest — COMMENTED** [review]
/approve
